### ISSUE TITLE
Build issues

### ISSUE THREAD

**Issue by ItachiSan** [TelepathyQt/telepathy-morse#64]
Hello,
I am the maintainer of the ArchLinux unofficial package `telepathy-morse-git`, which builds the package based on the latest sources (latest Git commit on the `master` branch).

I am currently having the following issue, when building:

```
$ make
[  9%] Automatic MOC for target telepathy-morse
[  9%] Built target telepathy-morse_autogen
Scanning dependencies of target telepathy-morse
[ 18%] Building CXX object CMakeFiles/telepathy-morse.dir/main.cpp.o
[ 27%] Building CXX object CMakeFiles/telepathy-morse.dir/connection.cpp.o
/tmp/telepathy-morse-git/src/telepathy-morse/connection.cpp:30:10: fatal error: TelegramQt/CTelegramCore: No such file or directory
 #include <TelegramQt/CTelegramCore>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/telepathy-morse.dir/build.make:76: CMakeFiles/telepathy-morse.dir/connection.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:73: CMakeFiles/telepathy-morse.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
```

What I did was to make a build directory alongside the source directory, run `cmake` within it then `make`.

I would like to understand which version of `telegram-qt` is actually required for properly building the package.

**Comment by a-andreyev** [comment]
Hello! :)
*next* branches (`source=("git+https://github.com/.../$_gitname#branch=next")`) for both *telepathy-morse* and *telegram-qt* are active branches and compiling fine for me.
You are probably building default git branches for both projects while only *telegram-qt* is currently have *next* branch as default. Combining *next* with *master* is not possible for now, so the solution could be to specify same branches directly via `PKGBUILD`.
You also could join [Telepathy Morse telegram group](https://t.me/Telepathy_Morse) if it's convenient for you to be in touch :)

**Comment by Kaffeine** [comment]
Thanks @a-andreyev :)

@ItachiSan, thank you for the packaging, I appreciate it.
I guessed that you use default branch instead of `master`.  The development takes place in the `next` branch and some github visitors asked me why the project has no commits in months, so I changed telegram-qt default branch to `next` about two weeks ago.
At first I wanted to 'fix' your issue by changing telepathy-morse default branch to `next` too, but now I doubt that the first version does work for anyone so it should be OK to merge `next` into `master` in both repositories and prepare 0.2.0-alpha1 releases. I'll do it this evening if you have no objections (like "Hey, I use 0.1.0 and it works for me").
